torchprime/hf_models/configs/default.yaml

```diff
@@ -0,0 +1,35 @@
+defaults:
+  - _self_
+
+# extra env variables you want to pass to the training script
+env:
+  - NUM_SLICE: 1
+  - XLA_PERSISTENT_CACHE_PATH: "/tmp/xla_cache"
+  - PROFILE_LOGDIR: "gs://torchprime/profile"
+  - XLA_USE_SPMD: 1
+  - ENABLE_PJRT_COMPATIBILITY: true
+  - PROFILE_DURATION_MS: 100000
+
```

Comment:
We could add a `profile_dir` config similar to https://github.com/AI-Hypercomputer/torchprime/blob/main/torchprime/torch_xla_models/configs/default.yaml#L21 -- this value will become what the user sets in `tp use`, followed by some date and slice ID folders etc.